thoth/metrics_exporter/jobs.py

```diff
@@ -82,16 +82,55 @@ def countGraphSyncJobStatus(JobListItems: dict) -> (int, int, int):
 
 def get_thoth_graph_sync_jobs():
     """Get the total number of graph-sync Jobs per category with corresponding status."""
-    namespace = os.getenv("MY_NAMESPACE")
+    namespaces = []
+    namespaces.append(os.getenv("MY_NAMESPACE"))
```

Comment:
Is that `THOTH_FRONTEND_NAMESPACE`?

Reply:
while you are looking at it... shall we move the metrics exporter from frontend to some other namespace? I think users dont need to access it.

Reply:
I think naming "frontend" is misleading in overall. Not sure if introducing new namespace would be a good option.

Reply:
MY_NAMESPACE is usually metadata.name in environment variables, so it depends where we deploy metrics-exporter. Also I'll add a check of the namespaces to analyze, because in thoth-test-core it would check three times the same namespace in that way.

Reply:
> MY_NAMESPACE is usually metadata.name in environment variables

Can it be explicit in the openshift template then?

> Also I'll add a check of the namespaces to analyze, because in thoth-test-core it would check three times the same namespace according to the current logic.

+1